internal/bootstrapping/agent.go

```diff
@@ -357,27 +357,29 @@ func (a *Agent) HandleResponse(inboxMsg *message.Message) ([]*message.Message, e
 	return nil, err
 }
 
-func (a *Agent) handleResponseMessage(msg *message.Message) (string, *MessageError, error) {
+func (a *Agent) handleResponseMessage(msg *message.Message) (string, *MessageError, bool, error) {
```

Comment:
`string, *MessageError, bool, error` - is it the right order? There are: string, msg error, bool, error. The boolean is somehow like a gap between the two errors.